src/Resources/views/Invoice/invoice.html.twig

```diff
@@ -0,0 +1,145 @@
+{% set shopBillingData = invoice.shopBillingData %}
+{% set additionalColspan = 0 %}
+{% if parrentTypeTemplate == 'show' %}
```

Comment:
I mentioned it before: `parrentTypeTemplate` is far too technical and written with a typo 😄 https://github.com/Sylius/InvoicingPlugin/pull/234/#discussion_r672850339